key,default_value
userAgent,"Boto3/1.9.133 Python/2.7.17 Linux/4.15.0-74-generic Botocore/1.12.133"
eventID,"e4a6e16d-2dc3-4ff5-9280-979cb035819"
errorMessage,"User: arn:aws:iam::811596193553:user/backup is not authorized to perform: cloudfront:ListCloudFrontOriginAccessIdentities"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"179.0.3.1"
eventName,"ListCloudFrontOriginAccessIdentities2018_11_05"
eventSource,"cloudfront.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"7db045d5-f059-4c92-bb11-f01d0ad5d612"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2020-02-25T14:41:13Z"
